src/ate/ate_perso_blob.cc

```diff
@@ -0,0 +1,405 @@
+// Copyright lowRISC contributors (OpenTitan project).
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
```

Comment:
perhaps you could drop a comment here on where this comes from? and file an issue to clean this up by bringing this in as a dep from the opentitan repo?